boolisslow.py

```diff
@@ -0,0 +1,40 @@
+"""
+True and False are keywords in python3, but not in python2. In python2
+the value of True is 1, and the value of False is 0. True and False can be
+regarded as vars and thus they are slow in while loop.
+
```

Comment:
Please add an explicit hint that this leads to a performance improvement only in Python2.